File: src/commands/utility/guilds.js
const Discord = require("discord.js");
const util = require("../../util");

module.exports = {
  name: "guilds",
  aliases: ["servers"],
  description: "Shows how many guilds the bot is in.",
  
  async execute(message, args, client) {
    const embed = new Discord.MessageEmbed()
      .setDescription(`${client.user.username} is currently in ${client.guilds.cache.size} servers with a total of ${client.guilds.cache.reduce((count, guild) => count + guild.memberCount, 0)} users.`)
      .setColor(util.color.blue);

    await message.reply({ embeds: [embed] });
  },
};

File: cogs/guilds.py
from discord.ext import commands, bridge
from discord import Embed
from utils import Utils
class Guilds(commands.Cog):
    def __init__(self, bot):
        self.bot = bot

    @bridge.bridge_command(aliases=["servers"], description = "Shows how many guilds the bot is in.")
    async def guilds(self, ctx):
        embed = Embed()
        embed.description = f"{self.bot.user.name} is currently in {len(self.bot.guilds)} servers with a total of {sum(x.member_count for x in self.bot.guilds)} users."
        embed.color = Utils.Colors.blue
        await ctx.respond(embed=embed)

def setup(bot):
    bot.add_cog(Guilds(bot))
